### PR TITLE
WFLY-5290 Adding JGroups stack seems to freeze the UI

### PR DESCRIPTION
**Pull request by rhusar**
Jira
https://issues.jboss.org/browse/WFLY-5290

### CODE REVIEW

**Comment by rhusar** [comment]
Update: debugging seems to hint at bug in core, where the alias registry is not kicking in despite of the translation happens correctly in standalone.

**Comment by rhusar** [comment]
Update: requires WF core upgrade (CR7) including https://github.com/wildfly/wildfly-core/pull/1150

File: clustering/jgroups/extension/src/main/java/org/jboss/as/clustering/jgroups/subsystem/TransportResourceDefinition.java [hunk 1]
```diff
@@ -241,7 +243,50 @@ public PathAddress transform(PathAddress address) {
...
+                    if (i == target.size() - 1) {
+                        final ModelNode operation = aliasContext.getOperation();
+                        final String stackName;
+
+                        if (ModelDescriptionConstants.ADD.equals(Operations.getName(operation)) && operation.hasDefined("type")) {
+                            stackName = operation.get("type").asString();
+                        } else {
+                            Resource root = null;
+                            try {
+                                root = aliasContext.readResourceFromRoot(PathAddress.pathAddress(result));
```
Comment (bstansberry) [inline clustering/jgroups/extension/src/main/java/org/jboss/as/clustering/jgroups/subsystem/TransportResourceDefinition.java]: This shouldn't block this PR, but passing the 'false' param to aliasContext.readResourceFromRoot should work fine. You still get the data on what the child resources are named, and you save the cost of cloning all the children.

**Comment by bstansberry** [comment]
@kabir Can you have a look at this since you're the alias expert?

**Comment by bstansberry** [comment]
http://brontes.lab.eng.brq.redhat.com/viewLog.html?buildId=75746&tab=buildResultsDiv&buildTypeId=WF_WildFlyCoreIntegrationExperiments is a custom CI run of this branch plus Kabir's branch from https://github.com/wildfly/wildfly-core/pull/1153. I assume this PR is the one that needed to be done in full for Kabir's branch to work.

**Pull request merged**